server/helpers/pushNotificationHelper.js

```diff
@@ -31,50 +31,6 @@ function sendPushNotification(receiver, data, req, res) {  // eslint-disable-lin
 
     const push = new PushNotifications(settings);
 
-    // let data = {
-    //     title: 'My Orange Push Notification', // REQUIRED
-    //     body: 'BOOYAH', // REQUIRED
-    //     topic: 'com.amida.orangeIgnite', // REQUIRED for apn and gcm for ios
-    //     custom: {
-    //         sender: 'Orange'
-    //     },
-    // eslint-disable-next-line max-len
-    //     priority: 'high', // gcm, apn. Supported values are 'high' or 'normal' (gcm). Will be translated to 10 and 5 for apn. Defaults to 'high'
-    //     contentAvailable: true, // gcm for android
-    //     delayWhileIdle: true, // gcm for android
-    //     restrictedPackageName: '', // gcm for android
-    //     dryRun: false, // gcm for android
-    //     icon: '', // gcm for android
-    //     tag: '', // gcm for android
-    //     color: '', // gcm for android
-    //     clickAction: '', // gcm for android. In ios, category will be used if not supplied
-    //     locKey: '', // gcm, apn
-    //     bodyLocArgs: '', // gcm, apn
-    //     retries: 1, // gcm, apn
-    //     encoding: '', // apn
-    //     badge: 2, // gcm for ios, apn
-    //     sound: 'ping.aiff', // gcm, apn
-    //     // alert: {}, // apn, will take precedence over title and body
-    //     alert: 'My Orange Push Notification', // It is also accepted a text message in alert
-    //     titleLocKey: '', // apn and gcm for ios
-    //     titleLocArgs: '', // apn and gcm for ios
-    //     launchImage: '', // apn and gcm for ios
-    //     action: '', // apn and gcm for ios
-    //     category: '', // apn and gcm for ios
-    //     mdm: '', // apn and gcm for ios
-    //     urlArgs: '', // apn and gcm for ios
-    //     truncateAtWordEnd: true, // apn and gcm for ios
-    //     mutableContent: 0, // apn
-    //     threadId: '', // apn
-    //     expiry: Math.floor(Date.now() / 1000) + 28 * 86400, // seconds
-    // eslint-disable-next-line max-len
-    //     timeToLive: 28 * 86400, // if both expiry and timeToLive are given, expiry will take precedency
-    //     headers: [], // wns
-    //     launch: '', // wns
-    //     duration: '', // wns
-    //     consolidationKey: 'my notification' // ADM
-    // };
```

Comment:
I deleted this large commented out chunk. But if it was acting as documentation, let me know and we can add it back in.

Reply:
@ElijahGeorge ?